docs/ldml/tr35-keyboards.md

```diff
@@ -224,7 +225,14 @@ One may also extend the notion of the ISO layout to support keys that don't map
 
 ![keyboard layout example showing extension of ISO key numbering](images/androidKeyboard.png)
 
-If it becomes necessary in the future, the format could extend the ISO layout to support keys that are located to the left of the "00" column by using negative column numbers "-01", "-02" and so on, or 100's complement "99", "98",... -->
+If it becomes necessary in the future, the format could extend the ISO layout to support keys that are located to the left of the "00" column by using negative column numbers "-01", "-02" and so on, or 100's complement "99", "98",… -->
```

Comment:
This is needlessly speculative. No reason to to that. 
```suggestion
If it becomes necessary in the future, the format could extend the ISO layout to support keys that are located to the left of the "00" column. -->
```

Reply:
Note that this is in commented out text which should be removed (or fixed)

Reply:
The format no longer uses ISO layout numbers anyway